README.md

```diff
@@ -10,14 +10,8 @@ It's currently Beta software and is not yet an official release. Documentation f
 
 ### Documentation
 
-This library uses [documentation](http://documentation.js.org/) to generate jsdocs, but it's not published anywhere yet.
-To view documentation locally run:
-
-```
-./scripts/build && open docs/index.html
-```
-
-To view the HTTP API documentation along with the node example code, see [https://partner-docs.recurly.com/](https://partner-docs.recurly.com/).
+Docs and example code can be found here: [https://recurly.github.io/recurly-client-node](https://recurly.github.io/recurly-client-node).
```

Comment:
This link 404s for me, what's the deployment plan here?

Reply:
can't enable it until we merge this.